src/pages/resources/migration/android/migrate-to-3x.md

```diff
@@ -100,6 +100,48 @@ Some of the APIs available in previous major versions of the Mobile SDK for Andr
 
 The `registerExtension` API for each extension that was deprecated in the 2.x version of the mobile SDK has been removed in the 3.x version of the mobile SDK. See the [Update SDK initialization](#update-sdk-initialization) section for more details.
 
+#### MobileCore
+
+| Removed API | Alternative API |
+| :------------- | :-------------- |
+| MobileCore.start(AdobeCallback)| [MobileCore.registerExtensions](../../../home/base/mobile-core/api-reference.md#registerextensions) registers extensions and starts event processing by default |
+| MobileCore.dispatchEventWithResponseCallback(Event, AdobeCallbackWithError)| Use [MobileCore.dispatchEventWithResponseCallback(Event, long, AdobeCallbackWithError)](../../../home/base/mobile-core/api-reference.md#dispatch--dispatcheventwithresponsecallback) to explicitly specify a timeout |
+| MessagingDelegate | Refer to this [page](../../../edge/adobe-journey-optimizer/in-app-message/tutorials/messaging-delegate/) to migrate to PresentationDelegate |
+| InvalidInitException| This exception is no longer thrown by the SDK |
+
+##### Extension Development
+
+Core 3.0.0 is not binary compatible with extensions built using earlier versions. Third-party extension developers are required to recompile their extensions with this version of Core.
+
+| Removed API | Alternative API |
+| :------------- | :-------------- |
+| MobileCore.log(LoggingMode, String, String) | Use logging methods exposed via [com.adobe.marketing.mobile.services.Log](https://github.com/adobe/aepsdk-core-android/blob/v3.0.0-core/code/core/src/phone/java/com/adobe/marketing/mobile/services/Log.java) |
```

Comment:
use main branch instead of v3.0.0 which may be removed later - same for all usages

Reply:
These links are actually pointing to the [tag](https://github.com/adobe/aepsdk-core-android/releases/tag/v3.0.0-core) `v3.0.0-core` and not the dev-3.0.0 branch.